lib/jazzy/sourcekitten/Rakefile

```diff
@@ -5,10 +5,10 @@ task :prepare do
   Dir.glob('*.framework').each do |fw|
     p fw
     Dir.chdir(fw) do
-      basename = File.basename(fw, '*.framework')
-      version = Dir.glob('Versions/*').sort.last
```

Comment:
why not use this logic, in case there's ever something besides `A` in the versions dir?